py/desitarget/mock/io.py

```diff
@@ -33,211 +38,106 @@
 ENCODE_FILE_MASK   = 2**ENCODE_FILE_END - 2**ENCODE_ROW_END
 ENCODE_FILE_MAX    = ENCODE_FILE_MASK >> ENCODE_ROW_END
 
-
+C_LIGHT = 299792.458
```

Comment:
Minor: preferable to use `astropy.constants.c.to('km/s').value` or `scipy.constants.c/1000`.